cmake/linker/armlink/target.cmake

```diff
@@ -76,7 +76,7 @@ function(toolchain_ld_link_elf)
     ${TOOLCHAIN_LD_LINK_ELF_LIBRARIES_PRE_SCRIPT}
     --scatter=${TOOLCHAIN_LD_LINK_ELF_LINKER_SCRIPT}
     ${TOOLCHAIN_LD_LINK_ELF_LIBRARIES_POST_SCRIPT}
-    $<TARGET_OBJECTS:arch__arm__core__aarch32__cortex_m>
+    $<TARGET_OBJECTS:arch__arm__core__cortex_m>
```

Comment:
Why not changed here?

https://github.com/zephyrproject-rtos/zephyr/blob/b3fa879348adc4087a5e9ffc5c2d10bb3dc52b37/cmake/linker/armlink/target.cmake#L67-L72

Reply:
fixed, thanks for spotting